modin/pandas/test/test_io.py

```diff
@@ -76,6 +76,9 @@
 # Number of rows in the test file
 NROWS = DATASET_SIZE_DICT.get(TestDatasetSize.get(), DATASET_SIZE_DICT["Small"])
 
+# Files compression to extension mapping
+comp_to_ext = {"gzip": "gz", "bz2": "bz2", "xz": "xz", "zip": "zip"}
```

Comment:
Variables in the global scope must be in upper case.
```suggestion
COMP_TO_EXT = {"gzip": "gz", "bz2": "bz2", "xz": "xz", "zip": "zip"}
```